kartothek/io/dask/bag.py

```diff
@@ -146,6 +148,7 @@ def read_dataset_as_dataframe_bag(
         load_dataset_metadata=False,
         predicates=predicates,
         dispatch_by=dispatch_by,
+        npartitions=npartitions,
```

Comment:
I would rather go for `partition_size` instead of `npartitions` for the following reason:
- the dask workers are likely limited by memory, so you want to cut down the load per worker
- you're tasks are likely being to small / to big and you want to control the balance between overhead and worker load
- the number of metapartitions is data-dependend, the worker / overhead per worker should not